### ISSUE TITLE
Code is missing

### ISSUE THREAD

**Issue by misc0110** [emsec/riscv-ucode#2]
The paper "On the Design and Misuse of Microcoded (Embedded) Processors — A Cautionary Note" was presented in August last year, and already the abstract says "To foster future research and applications, we publicly release our implementation and evaluation platform.". The link points to this repository. However, there is no code here, it only says "Under construction" since January 2021. 

This is very disappointing. Please release the code as promised in the paper.

**Comment by nils1603** [comment]
I totally agree with you and I am extremely sorry about that. Right now our code-quality is not a publishable state and needs a lot of documentation and polishing. We did not yet find the time to do that yet.

**Comment by nils1603** [comment]
A quick update. We just finished another submission and are back from holiday. We will start working on this next week!

**Comment by TNdu** [comment]
Any update about the release of the source code ?
